### ISSUE TITLE
Wrong window.devicePixelRatio

### ISSUE THREAD

**Issue by TheCloudlessSky** [puppeteer/puppeteer#2372]
### Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 1.2.0
* Platform / OS version: Windows 7 Pro and Windows 10 Pro
* URLs (if applicable): N/A
* Node.js version: 8.10.0
* Local chromium version: win64-543305 (67.0.3372.0)
* Windows display DPI: 125%

**What steps will reproduce the problem?**

When I evaluate `window.devicePixelRatio` when the browser is first launched with `headless: false`, it reports an incorrect value (`1.0000000149011612`). Here's an example:

```
const puppeteer = require('puppeteer');
(async () => {
  let browser;
  try {
    browser = await puppeteer.launch({ headless: false });
    const page = await browser.newPage();
    let dpr = await page.evaluate('window.devicePixelRatio');
    console.log(dpr);
  }
  finally {
    if (browser) {
      await browser.close();
    }
  }
})();
```

**What is the expected result?**

To return 1.25.

**What happens instead?**

It returns 1.0000000149011612.

---

While the browser is open, if I open up the DevTools then reevaluate the `window.devicePixelRatio`, it reports the right value of 1.25:

```
const puppeteer = require('puppeteer');

(async () => {
  let browser;
  try {
    browser = await puppeteer.launch({ headless: false });

    const page = await browser.newPage();

    let dprBefore = await page.evaluate('window.devicePixelRatio');
    // Prints "1.0000000149011612".
    console.log(dprBefore);

    await page.waitFor(30 * 1000);

    // Now, open DevTools with F12.

    let dprAfter = await page.evaluate('window.devicePixelRatio');
    // Prints "1.25".
    console.log(dprAfter);
  }
  finally {
    if (browser) {
      await browser.close();
    }
  }
})();
```

I've also tried launching the original script with my global Chrome executable, but it still reports the wrong devicePixelRatio:

```
    browser = await puppeteer.launch({
      headless: false,
      // Version is 65.0.3325.181
      executablePath: 'C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe'
    });
```

I've also tried explicitly setting the `--force-device-scale-factor=X` argument. 

| --force-device-scale-factor  | window.devicePixelRatio |
| ------------- | ------------- |
| 1  | 1 |
| 1.25  | 1.0000000149011612  |
| 1.5 | 1.0000000298023224 |
| 2 | 1 |

When `headless: true`, the DPR is always set to 1. However, if I use `--froce-device-scale-factor`, `window.devicePixelRatio` returns the *same* as the previous table.

I've also tried launching the local Chromium instance (with the same arguments that Puppeteer would use to launch `chrome.exe`):

```
./node_modules\puppeteer\.local-chromium\win64-543305\chrome-win32\chrome.exe --disable-background-networking --disable-background-timer-throttling --disable-client-side-phishing-detection --disable-default-apps --disable-extensions --disable-hang-monitor --disable-popup-blocking --disable-prompt-on-repost --disable-sync --disable-translate --metrics-recording-only --no-first-run --safebrowsing-disable-auto-update --enable-automation --password-store=basic --use-mock-keychain --remote-debugging-port=0 --user-data-dir=C:\Temp\puppeteer_dev_profile-abc123 https://google.ca file:///C:/dpr.html
```

`C:\dpr.html`:

```
<!doctype html>
<html>
    <head><title>DPR</title></head>
    <body>
        <h1>
            <script>
                document.write(window.devicePixelRatio);
            </script>
        </h1>
    </body>
</html>
```

When this launches, the 2nd tab "DPR" shows the correct DPR:

![image](https://user-images.githubusercontent.com/613788/38752181-54a75e9a-3f28-11e8-8314-423bb4221a55.png)

My machine is Windows 10 Pro but I've also been able to reproduce this on another persons's Windows 7 Pro laptop with a DPI of 125%.

**Comment by aslushnikov** [comment]
@TheCloudlessSky pptr overrides default viewport with dpi = 1. We'll add an ability to disable default viewport override, see #1183.

**Comment by TheCloudlessSky** [comment]
Until the ability to disable the default viewport override is available, I'm using this trick from #1183:

```
await page._client.send('Emulation.clearDeviceMetricsOverride');
```
